test/libyul/yulOptimizerTests/fullSimplify/operations.yul

```diff
@@ -37,8 +38,9 @@
 //     mstore(12, 0)
 //     mstore(13, 0)
 //     mstore(14, 0)
-//     mstore(15, 0)
-//     mstore(16, 0)
-//     mstore(17, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
-//     mstore(18, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
+//     let _39 := not(x)
+//     mstore(15, and(x, _39))
```

Comment:
`and(x, not(x))` seems like a convoluted way to write `0` 😉

Reply:
weird, that should also be optimized away :(

Reply:
I guess it is the order of operations now.